content/content-strategy/manage-content-requests/plan-content/content-planning-workshop/section.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: text/textblock
+section: Why run a workshop
+---
+### [1]Materials
```

Comment:
Incorrect display level. Needs to be `####` to show a heading 4.

Reply:
After discussing made these methods headers a ## [2]